intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/build/BuildProgressReporter.xtend

```diff
@@ -57,7 +57,7 @@ class BuildProgressReporter implements BuildRequest.IPostValidationCallback {
 
 	override afterValidate(URI validated, Iterable<Issue> issues) {
 		markAsAffected(validated)
-		for (issue : issues) {
+		for (issue : issues.filterNull) {
```

Comment:
Would it be better if we prevent null-issues in the iterable in the first place?